core/src/main/scala/medeia/generic/HlistDecoderInstances.scala

```diff
@@ -14,18 +14,19 @@ trait HlistDecoderInstances {
 
   implicit def hlistObjectDecoder[Base, K <: Symbol, H, T <: HList](implicit
       witness: Witness.Aux[K],
-      hDecoder: Lazy[BsonDecoder[H]],
+      hDecoder: BsonDecoder[H],
```

Comment:
Nice that it works without it, but wasn't this important somehow?

Reply:
If the implicit resolution works, it should not be important :shrug:  Unless there is a case in a bigger codebase which we never tested :thinking:  We can try it or revert, your call :wink:

Reply:
We don't have it in `scynamo` and it never led to any problems

Reply:
We'll remove it and I will try remove the other occurrences that might be unneeded also